bundles/org.openhab.binding.siemenshvac/src/main/resources/OH-INF/thing/ozw672.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="siemenshvac"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bridge Thing Type -->
+	<bridge-type id="ozw672">
```

Comment:
I'm not really sure on that, as I don't have documentation about Ozw772, and don't have access to this physical device to do some test. But I except that the code should be identical between the two devices.

Today, in the binding, the only specific part is in bridge initialization / discovery where we test the modelName, and initialize an OZW672. Notes that in current configuration, we will not currently support Ozw772 because we don't have an SiemensHvacOZW772BridgeThingHandler override the SiemensHvacBridgeBaseThingHandler, and so we have no way to init the addon for this device.

So I think we can go two way :

- Scenario 1: Keep like this for first version of the addon.
- Scenario 2: extends the discovering and bridge initialization to handle OZW772, adding an SiemensHvacOZW772BridgeThingHandler override, and see if someone has the device to do some testing.

Reply:
It sounds like you would expect OZW772 to be similar rather than different. The implementation and support can always be changed, but the modelling cannot without breaking backwards compatibility.

So I would propose to keep the current implementation, but generalize the bridge type id: `ozw672` → `ozw`. I think in the long run it will then be easier to add support for OZW772 with less duplication (i.e. separate bridge definitions, handlers, etc.). It won't hurt to also rename `SiemensHvacOZW772BridgeThingHandler` etc., but strictly it's not necessary as it can always be refactored without breaking anything.

Reply:
It seems I was not able to convince you about consolidating into thing type `ozw`, since you went the other way and introduced another thing type `ozw772`. You will now need to run `mvn i18n:generate-default-translations` to provide the redundant configuration parameter labels etc. for the two bridge types.

Reply:
generation of i18n done, fix on next commit

Reply:
Finally, I move back on this one.
I review my point of view, and align to your recommandation to have only one class for the bridge.